package.json

```diff
@@ -80,23 +83,23 @@
     "glotpress": "gulp glotpress",
     "changelog": "./vendor/bin/changelogger add"
   },
-  "engines": { "node": "18.13.0", "npm": "8.19.3" },
+  "engines": {
+    "node": "18.17.0",
+    "npm": "9.6.7"
+  },
   "devDependencies": {
```

Comment:
We should alphabetize these - reduces "noise" in changes.

Reply:
The noise comes from the sync with taskmaster, that will update the `package.json` file on each `npm install` call.